puppet/zulip/templates/memcached.conf.centos.template.erb

```diff
@@ -0,0 +1,5 @@
+PORT="11211"
+USER="memcached"
+MAXCONN="1024"
+CACHESIZE="<%= @memcached_memory %>"
+OPTIONS="-l 127.0.0.1 -S -vv"
```

Comment:
I think the `-vv` is a conversion bug; looks like you translated a commented line.

Reply:
`-vv` removed.